lib/microservices_router.rb

```diff
@@ -0,0 +1,12 @@
+require 'net/http'
+require 'uri'
+
+module MicroservicesRouter
```

Comment:
Alright. Is it a service change when the service is the same but responsible for changed models?